app/main.py

```diff
@@ -1,6 +1,34 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Kyiv"
+API_KEY = os.environ.get("WEATHER_API_KEY")
```

Comment:
In this task you will be implementing the service for getting current weather for ```Paris``` city.